### ISSUE TITLE
[Makefile:150: help] Error 127

### ISSUE THREAD

**Issue by Etisti** [drivendataorg/sfp-cervical-biopsy-runtime#18]
Hi, i'm trying to use the makefile (I'm on windows). 

I tryed first with wsl2 then i got path error 127 when using 'make sample-images' or 'make pack-benchmark', and i also got an error concerning a removing file when using the 'make test-container'.

Then i tryed with gnuwin, mingw, chocolatey and got the same error (image below) i cannot resolve ><

![image](https://user-images.githubusercontent.com/42436475/94062001-6eb6ca80-fde6-11ea-905f-08115f2b5de9.png)

If someone can help me with this issue , thanks 🙏

**Comment by pjbull** [comment]
Unfortunately, Windows support for the Makefile is tough since there are many different variants of installing GNU make and a shell like bash on Windows. I'd happily accept a `.bat` or equivalent version of the make commands that just works at the windows command prompt.

That said, I did some testing on Windows (using Git Bash, in case that is helpful). I only have a VM to test on, so I couldn't install docker. With the change in #19 and installing `zip` through [GoW](https://github.com/bmatzelle/gow/releases), I was able to get `make`, `make sample-images`, and `make pack-benchmark` to complete successfully (though it will print some errors about `nvidia-smi` and `docker`).

Could you confirm and let me know if the docker commands work when you have docker installed?

If so, I'd like to close this issue and open a new one for a native Windows version of the helper commands. Hopefully someone in the community can help with that.

**Comment by pjbull** [comment]
Hi @Etisti, just checking if the current version runs for you on Windows. Thanks!

**Comment by Etisti** [comment]
Hi, "make" gives me thoses responses : 
"make pull" works for make with minGW and wsl.
"make build" freeze for both minGW and wsl at removing tflow :
 
![image](https://user-images.githubusercontent.com/42436475/94561473-59b5bd80-0264-11eb-819b-321a7b7367ef.png)

for "make" with minGW :

![image](https://user-images.githubusercontent.com/42436475/94550266-bfe61480-0253-11eb-88b8-c6852858e8f1.png)

"make sample-images" and "make pack-benchmark" with minGW worked 👍 

"make pack-benchmark" with minGW didn't : 
![image](https://user-images.githubusercontent.com/42436475/94550971-e35d8f00-0254-11eb-87af-ba166cc31305.png)

for "make" with wsl :
![image](https://user-images.githubusercontent.com/42436475/94550377-e906a500-0253-11eb-802a-8aada8cfe9ab.png)

"make sample-images" with wsl :
![image](https://user-images.githubusercontent.com/42436475/94550525-2703c900-0254-11eb-8062-8360986631b0.png)

"make pack-benchmark" with wsl : 
![image](https://user-images.githubusercontent.com/42436475/94550629-561a3a80-0254-11eb-8cb4-4372a7c222dd.png)
(seems with wsl there is a path problem)

**Comment by pjbull** [comment]
>  "make build" freeze for both minGW and wsl at removing tflow :

I don't think this is a Windows problem. It may be specific to the CPU/RAM/disk space on your particular machine. The dependencies here are heavy, and building the container is always going to require a relatively powerful machine.

There seem to be three other issues that you are seeing:
 - The version of `sort` in these environments [does not have an `--ignore-case` flag](https://github.com/drivendataorg/sfp-cervical-biopsy-runtime/blob/61ac6b158762c2bfe1a2e57d66675c9648180ddb/Makefile#L164). We may just be able to remove that since the sorting is a "nice to have"
 - The executable `zip` is not found. As I mentioned, this can be fixed by installing zip through [GoW](https://github.com/bmatzelle/gow/releases) or similar. I will add a note to the prereqs about installing this on Windows.
 - The executable `aws` is not found. Needing the AWS CLI installed and configured is already mentioned in the prerequisites: https://github.com/drivendataorg/sfp-cervical-biopsy-runtime#prerequisites

**Comment by Etisti** [comment]
Aws cli is already installed and worked for the make sample images command (not the wsl but with minGW).
And I don't think it is machine related because I have still 25GB left, and that I also tryed the make commands on my Linux partition and they worked fine.
